### PR TITLE
Archie/refactor flurry

### PR DESCRIPTION
**Pull request by archiepeterjarvis**
# Archie/refactor flurry

[Jira Ticket](https://yetitoolsmartsoftware.atlassian.net/jira/software/projects/YTSS/boards/1?assignee=5f8db15e6bc6340068c0bfc3&selectedIssue=YTSS-2970)
- [x] Ready for review

## Checklist
- [x] I have completed a self review
- [ ] I have set the recent milestone
- [ ] I have tested graphical changes on all languages
- [ ] I have updated the jira ticket
- [x] I have added the relevant labels to this PR
- [ ] I have updated documentation (if applicable)
- [ ] I have run the unit tests suite and they pass
- [ ] I have updated the [change log](https://docs.google.com/document/d/1t3s6lQuUcug1Fj8E_i2gBJNvXN0-funwBArG2hsia6k/edit)

## Description
This branch introduces a new system for Flurry, which is a bit neater and refined overall. It DOES NOT migrate all previous functionality (like jobs, current machine status etc). This is just for harvesting 'persistent' data (like machine info, grbl settings, etc), and the other system will be replaced in the future once we decide on strategy.

## Testing
I did some basic test cases to test this, here's the [doc](https://docs.google.com/document/d/18qrNudc6bm3BN54vZOJ6egPOaeG09WsDpdxeplPyai4/edit)

### Visual Test
- [x] Not applicable
- [ ] Own computer
- [ ] Console 7"
- [ ] Console 10"

### Function Test
- [ ] Not applicable
- [x] Own computer
- [x] Console 7"
- [ ] Console 10"

### Unit Tests
- [ ] Not applicable
- [ ] Completed

## Screenshots (if applicable)

### CODE REVIEW

**Review by Masc999 — CHANGES_REQUESTED** [review]
Please don't delete migration.json!!! (Only I am allowed to do so and complain afterwards...) 😄

File: src/asmcnc/comms/serial_connection.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ class SerialConnection(EventDispatcher):
     setting_1 = NumericProperty(-1.0)
     setting_2 = NumericProperty(-1.0)
     setting_4 = NumericProperty(-1.0)
+    setting_3 = NumericProperty(-1.0)
```
Comment (Masc999) [inline src/asmcnc/comms/serial_connection.py:46]: Almost the right order... ;)

File: src/main.py [hunk 2]
```diff
@@ -198,12 +204,14 @@ def build(self):
 
         # Initialise settings object
         sett = settings_manager.Settings(sm)
```
Comment (Masc999) [inline src/main.py:206]: Why initialise "sett" at all? just use self.settings_manager. In the long run, objects should get their things through app.get_running_app anyways.

File: src/main.py [hunk 3]
```diff
@@ -198,12 +204,14 @@ def build(self):
...
+        self.settings_manager = sett
 
         # Initialise 'j'ob 'd'ata object
         jd = job_data.JobData(localization=self.l, settings_manager=sett)
 
         # Initialise 'm'achine object
         m = router_machine.RouterMachine(Cmport, sm, sett, self.l, jd)
+        self.machine = m
```
Comment (Masc999) [inline src/main.py:214]: same here for m.

Comment (archiepeterjarvis) [reply src/main.py:214]: It was quicker than renaming all references to self.machine 🤣

**Review by Masc999 — APPROVED** [review]

**Pull request merged**